src/OpenTok/OpenTok.php

```diff
@@ -478,6 +478,58 @@ public function forceDisconnect($sessionId, $connectionId)
         return $this->client->forceDisconnect($sessionId, $connectionId);
     }
 
+    /**
+     * Mute a stream in an OpenTok session
+     *
+     * @param string $sessionId The OpenTok session ID that the client is connected to
+     *
+     * @param string $streamId The OpenTok streamId to be muted in the session
+     *
+     * @return bool
+     */
+    public function muteStreamInSession(string $sessionId, string $streamId): bool
```

Comment:
This does not match the proposed API (`OpenTok.forceMuteStream()`. See https://github.com/opentok/api-review/tree/master/mute-all-api.